src/cmd-run

```diff
@@ -222,6 +236,20 @@ if [ "$(arch)" == "aarch64" ]; then
     set -- -bios /usr/share/AAVMF/AAVMF_CODE.fd "$@"
 fi
 
+if [ "$UEFI" == "1" ]; then
+    cp /usr/share/edk2/ovmf/OVMF_VARS.fd /tmp/vars.fd
```

Comment:
Hmm, then those files stick around even after `cosa run`, right? Can we use the same trick here as we do for `VM_IMG`? E.g. `exec 7<> /tmp/OVMF_VARS.fd && rm -f /tmp/OVMF_VARS.fd`, then `-drive file=/proc/self/fd/7,...`.

Reply:
SGTM fixed. Accidently deleled the branch. Fixed version here: https://github.com/coreos/coreos-assembler/pull/570